safe_transaction_service/history/services/notification_service.py

```diff
@@ -204,3 +205,54 @@ def build_reorg_payload(block_number: int) -> ReorgPayload:
         blockNumber=block_number,
         chainId=str(get_chain_id()),
     )
+
+
+class DelegatePayload(TypedDict):
+    type: str
+    safeAddress: Optional[str]
+    delegate: str
+    delegator: str
+    label: str
+    expiryDate: Optional[int]
```

Comment:
Maybe add the unit, for clarity. Like `expiryDateSeconds`

Reply:
Done https://github.com/safe-global/safe-transaction-service/pull/2310/commits/e90cc8554af0d1d6fb5ada379215ba698099c7b2